features/_secureboot/test/test_uefi_pki_files.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from helper.utils import execute_remote_command
+
+
+@pytest.mark.parametrize(
+    "pki_file",
+    [
+            "/etc/gardenlinux/gardenlinux-secureboot.pk.auth",
+            "/etc/gardenlinux/gardenlinux-secureboot.kek.auth",
+            "/etc/gardenlinux/gardenlinux-secureboot.db.auth"
+    ]
+)
+
+
+def test_uefi_pki_files(client, pki_file):
+    # This will already fail if file is absent
+    execute_remote_command(client, f"stat {pki_file}")
```

Comment:
Maybe we should enhance the error message and not just pass the error message of the command.